wfe/jose_test.go

```diff
@@ -59,6 +60,11 @@ func TestRejectsHS256(t *testing.T) {
 }
 
 func TestCheckAlgorithm(t *testing.T) {
+	key, err := rsa.GenerateKey(rand.Reader, 2048)
```

Comment:
Thanks for fixing this issue with the tests! Could you instead generate a pubkey once and save it in testdata? That will avoid spending extra time to generate a key with each test run.